src/job-feed/__tests__/JobList.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {
+  screen,
+} from '@testing-library/dom';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { MockedProvider } from '@apollo/client/testing';
+import JobList from '../JobList';
+import { GET_JOBS } from '../../sources';
+
+const mocks = [{
+  request: {
+    query: GET_JOBS,
+  },
+  result: {
+    data: {
+      jobs: [
+        {
+          id: 1,
+          name: 'Developer',
+          description: 'Very good developer',
+          company: {
+            name: 'CoolCompany',
+          },
+          jobSkills: [
+            {
+              skill: {
+                id: 1,
+                name: 'ReactJS',
+              },
+            },
+          ],
+        },
+      ],
+    },
+  },
+}];
+
+describe('tests for JobList', async () => {
+  beforeEach(async () => {
+    render(
+      <MockedProvider mocks={mocks}>
+        <JobList />
+      </MockedProvider>,
+    );
+  });
+
+  it('should see job details on the page', async () => {
```

Comment:
I don't see much difference between this tests and the one for the job card. Here I would test that all the N jobs are displayed. We should care here about the list, not what an entry in that list might contain (already tested in another suite).

Reply:
updated